R/loo_compare.R

```diff
@@ -131,9 +154,10 @@ loo_compare.default <- function(x, ...) {
 #' @param digits For the print method only, the number of digits to use when
 #'   printing.
 #' @param simplify For the print method only, should only the essential columns
-#'   of the summary matrix be printed? The entire matrix is always returned, but
-#'   by default only the most important columns are printed.
-print.compare.loo <- function(x, ..., digits = 1, simplify = TRUE) {
+#'   of the summary matrix be printed? The entire matrix is always returned, bu#' @param pnorm For the print method only, should we include the normal
+#'   approximation based probability of model having worse performance than
+#'   the best model
```

Comment:
The end of doc for the `simplify` argument got deleted and `#' @param pnorm` should be on a new line. 

```suggestion
#'   of the summary matrix be printed? The entire matrix is always returned, but
#'   by default only the most important columns are printed.
#' @param pnorm For the print method only, should we include the normal
#'   approximation based probability of each model having worse performance than
#'   the best model?
```